### DISCUSSION TITLE
how to support a new mysql compatible database

### DISCUSSION THREAD

**Discussion by zhjwpku** [sqlalchemy/sqlalchemy#7968] [Usage Questions] [answered]
I want to support a new database which is compatible with mysql(but
not 100% compatible),
I am wondering if there are any test suit that can make sure if the
tests are all passed, sqlalchemy will work for that database?

**Comment by CaselIT** [comment, marked as answer]
Hi,

There are two readme files in the repository that talk about unittest and dialect development.

Sqlalchemy has an acceptance suite that dialects can use to test their logic against a running db.

It may also be useful to examine other external dialects, such as sqlalchemy-firebird, sqlalchemy-access or sqlalchemy-sybase

**Reply by zhjwpku** [reply]
Thanks for the quick response, will take a look ;)
